code/game/objects/items/weaponry.dm

```diff
@@ -223,13 +223,51 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
 	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 50)
 	resistance_flags = FIRE_PROOF
 
-/obj/item/katana/cursed
-	slot_flags = null
-
 /obj/item/katana/suicide_act(mob/user)
 	user.visible_message("<span class='suicide'>[user] is slitting [user.p_their()] stomach open with [src]! It looks like [user.p_theyre()] trying to commit seppuku!</span>")
 	return(BRUTELOSS)
 
+/obj/item/katana/cursed
+	name = "ominous katana"
+	desc = "An ancient single-edged weapon, possessing a flawless sheen despite it's clear age. The blade seems almost too perfect, and passes through most substances like water. <span class='warning'><b>Something is wrong.</b></span>."
+	force = 50
```

Comment:
Far too much damage, reduce it and make the faction bonus larger

Reply:
I suggest `force = 20` and `faction_bonus_force = 30` to still meet that 50 damage bar while keeping it on-par with other swords.